src/test/java/systemtests/AddCommandSystemTest.java

```diff
@@ -105,34 +105,7 @@ public void add() {
                 .withDepth(40)
                 .build();
         assertCommandFailure(DiveUtil.getAddCommand(diveSession), (Messages.MESSAGE_ERROR_LIMIT_EXCEED + " " + HARDCODE_MAX_TIME));
-
-        /*
-         * Case: Dive Depth is invalid - zero
-         */
-//        diveSession = new DiveSessionBuilder()
```

Comment:
Why is this removed!!!

Reply:
Eh, I added that, but then there was other checks in place that would have made it harder thats why I inserted and then removed it